Dockerfile-centos-7

```diff
@@ -3,10 +3,14 @@ FROM centos/systemd
 ARG FPM_VERSION=1.11.0
 ARG PYTHON_VERSION=3
 ARG DOCKER_WORKDIR=/usr/share/pdagent
+ARG SKIP_SYSTEMD=false
 ENV PYTHON_VERSION ${PYTHON_VERSION}
 ENV container docker
 ENV DOCKER_WORKDIR ${DOCKER_WORKDIR}
 
+# There is known issues with Docker + systemd
```

Comment:
```suggestion
# There are known compatibility issues with Docker + systemd -- they both assume they are running with pid 1
```

plus maybe a handy link if we have it?

Reply:
i can't judge whether we do need this much detail.......

Reply:
Eh: I feel like it's helpful to include the concrete reason why, it's not a trade secret or anything.